articles/TOC.yml

```diff
@@ -12,25 +10,31 @@
     href: install-guide/update.md  
   - name: Write your first quantum program
     href: quickstart.md
-- name: Managing quantum machines and drivers
-  href: machines/toc.yml
-- name: Quantum development techniques
-  topicHref: techniques/index.md
-  href: techniques/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Q# programming language
-  topicHref: language/index.md
-  href: language/toc.yml
-- name: Contributing to the Quantum Development Kit
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: Glossary
-  href: Glossary.md
-- name: For more information
-  href: for-more-info.md
-- name: Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
-- name: License
-  href: license.md
+- name: Concepts
+  items:
+  - name: Quantum computing concepts
+    href: concepts/toc.yml
+  - name: Quantum development techniques
+    topicHref: techniques/index.md
+    href: techniques/toc.yml
+  - name: Q# programming language
+    topicHref: language/index.md
+    href: language/toc.yml
+  - name: How to host a quantum machine
+    href: machines/toc.yml
+  - name: Q# libraries
```

Comment:
Similarly, I'd suggest moving this above `machines/*`, as I would expect (and hope!) that using the libraries is a more basic concept than customizing classical host programs.